packages/nc-gui/app.vue

```diff
@@ -1,13 +1,24 @@
 <script setup lang="ts">
-import { applyNonSelectable, computed, useRoute, useTheme } from '#imports'
+import { computed, useRoute, useTheme } from '#imports'
 
 const route = useRoute()
 
 const disableBaseLayout = computed(() => route.path.startsWith('/nc/view') || route.path.startsWith('/nc/form'))
 
 useTheme()
 
-applyNonSelectable()
+useEventListener(document, 'keydown', async (e: KeyboardEvent) => {
+  const cmdOrCtrl = isMac() ? e.metaKey : e.ctrlKey
+  if (cmdOrCtrl) {
+    switch (e.code) {
+      case 'KeyA':
```

Comment:
Same concern with other comment but just to note while using e.key we should also consider case as it can be 'a' or 'A' (depending on caps-lock state or shift modifier)